README.md

```diff
@@ -1 +1,151 @@
-# audit-service-deploy
\ No newline at end of file
+## Audit Service Deployment
+
+### Introduction
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  This definition is registered with the Audit service and used to create a Java SDK to record the audited events.  The Audit service is multi-tenant aware, which requires an application producing events to register each new tenant.  Applications send events using the generated Java SDK to Elasticsearch where they are Indexed according to tenant.
+
+### Deployment Repository
+
+This repository provides the necessary files to easily get started using the Audit Service.
+
+The only pre-requisite required to get started is that Docker must be available on the system.
+
+The deployment files are in Docker Compose v3 format, and they are compatible with both Docker Compose and Docker Stack.
+
+The deployment file references at present only references Elasticsearch.
+
+### Demonstration
+
+The Docker Compose file contains the following services:
+
+
+<table><img src=./images/audit_service_deploy.jpg /></table>
+
+**Elasticsearch**  
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) is a search engine based on [Lucene](https://lucene.apache.org/core/). It provides a distributed, multitenant-capable full-text search engine with an HTTP web interface and schema-free JSON documents. Elasticsearch is developed in Java and is released as open source under the terms of the Apache License.
```

Comment:
https://www.quora.com/A-http-or-an-http-which-is-the-correct-grammatical-usage

Should be "an"

Note that "http" is sometimes called an acronym but properly it's an initialism not an acronym because nobody pronounces it as a word, they pronounce it as a series of letters: h-t-t-p. Therefore, whether you use "a" or "an" (and like whether you pronounce "the" as "thuh" or "thee") depends on how you pronounce "h". The prestigious pronunciation is "aytch", which starts with a vowel, so "an". A significant minority pronounce it "haytch" and if you feel you need to copy that to blend in where you live, use "a".